test/e2e/framework/framework.go

```diff
@@ -448,11 +451,9 @@ func (f *Framework) CreateNamespace(ctx context.Context, baseName string, labels
 		labels = labelsCopy
 	}
 
```

Comment:
Is this true? If if is, maybe add it.
```suggestion

// You can set a single attribute (NamespacePodSecurityLevel) that configures all three pod security
// admission labels. You can also specify the enforce, warn, or audit levels individually, overriding
// the general level if set. The default is to use "restricted".
//
// The E2E framework then sets those pod security labels on the namespace.
```

Reply:
added more related documentation to the framework struct